core/core-api/src/main/java/com/hbc/pms/core/api/support/data/AlarmStore.java

```diff
@@ -0,0 +1,47 @@
+package com.hbc.pms.core.api.support.data;
+
+import com.hbc.pms.core.model.AlarmCondition;
+import com.hbc.pms.plc.api.IoResponse;
+import org.springframework.stereotype.Component;
+
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Component
+public class AlarmStore {

```

Comment:
Please update detailed design for this class, this class looks like one of the main class for the Alrm flow